### PR TITLE
Model changes from 20230412 discussion

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

**Review by mbvertex — DISMISSED** [review]

**Review by therealsilence — COMMENTED** [review]
Few minor things

File: synapse/models/risk.py [hunk 1]
```diff
@@ -201,12 +226,13 @@ def getModelDefs(self):
...
                 )),
+                ('risk:vuln:type:taxonomy', {}, ()),
                 ('risk:vuln', {}, (
                     ('name', ('str', {}), {
                         'doc': 'A user specified name for the vulnerability.'}),
 
-                    ('type', ('str', {}), {
-                        'doc': 'A user specified type for the vulnerability.'}),
+                    ('type', ('risk:vuln:type:taxonomy', {}), {
+                        'doc': 'A taxonomy type entry for the vulnerability.'}),
```
Comment (Cisphyx) [inline synapse/models/risk.py:235]: Need a migration for `risk:vuln:type:taxonomy`

File: synapse/models/person.py [hunk 2]
```diff
@@ -390,6 +390,9 @@ def getModelDefs(self):
                     ('place', ('geo:place', {}), {
                         'doc': 'The place associated with this contact.',
                     }),
+                    ('place:name', ('geo:name', {}), {
+                        'doc': 'The reported place name of the contact.',
```
Comment (Cisphyx) [inline synapse/models/person.py:394]: sounds a little odd to me, maybe "reported name of the place associated with this contact" maybe?

File: synapse/models/inet.py [hunk 3]
```diff
@@ -2375,17 +2385,66 @@ def getModelDefs(self):
                             'doc': 'The name of the file owned by or associated with the account.'
                         }),
                         ('posted', ('time', {}), {
-                            'doc': 'The date and time the file was posted / submitted.'
-                        }),
+                            'deprecated': True,
```
[the remark this answers is not in the document]
Comment (invisig0th) [reply synapse/models/inet.py:2388]: The "fused" knowledge ( as the comp type essentially enforces ) :+1:

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**